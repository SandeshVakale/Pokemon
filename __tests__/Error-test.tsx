import 'react-native';
import React from 'react';
import Error from '../src/components/error';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders error correctly', async () => {
  const tree = renderer
    .create(
      <Error
        error={{
          isError: true,
          message: 'Something went wrong',
          code: '009',
        }}
      />,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
