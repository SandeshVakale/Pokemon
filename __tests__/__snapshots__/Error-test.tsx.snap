// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders error correctly 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text />
  <Text
    style={
      Object {
        "fontFamily": "Lexend_700Bold",
        "fontSize": 24,
      }
    }
  >
    009
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "Lexend_400Regular",
        "fontSize": 20,
      }
    }
  >
    Something went wrong
  </Text>
</RCTSafeAreaView>
`;
